Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+    <img src="./Img/corisLogo.png">
+</p>
+
+# <div style="text-align: right; text-align: down"> Mode d'emploi </div>
+# <div style="text-align: right; text-align: down"> Version 1.1 </div>
+
+<div style="page-break-after: always;"></div>
+
+<details open>
+<summary>Sommaire</summary>
+
+- [ Mode d'emploi ](#-mode-demploi-)
+- [ Version 1.1 ](#-version-11-)
+- [Mesures de sécurité](#mesures-de-sécurité)
+  - [Avertissements](#avertissements)
+- [Premières Etapes](#premières-etapes)
+  - [Contenu de la boîte](#contenu-de-la-boîte)
+  - [Chargez votre SportShield](#chargez-votre-sportshield)
+  - [Initialisation de votre SportShield](#initialisation-de-votre-sportshield)
+- [Comment verrouiller votre SportShield](#comment-verrouiller-votre-sportshield)
+- [Comment déverrouiller votre SportShield](#comment-déverrouiller-votre-sportshield)
+- [Alarme](#alarme)
+- [App](#app)
+- [Notifications](#notifications)
+- [Informations Générales and Spécifications techniques](#informations-générales-and-spécifications-techniques)
+  - [Composants](#composants)
+  - [Garantie de robustesse](#garantie-de-robustesse)
+    - [Câble d'antivol](#câble-dantivol)
+  - [Conditions environnementales](#conditions-environnementales)
+  - [Sécurité](#sécurité)
+  - [Glossaire](#glossaire)
+</details>
+
+<br>
+
+<div style="page-break-after: always;"></div>
+
+# Mesures de sécurité
+
+> ⚠️ ATTENTION • Avant utilisation, lisez le mode d'emploi pour vous assurer d'une utilisation correcte et d'une bonne compréhension. Après lecture, conservez-le dans un endroit sûr pour pouvoir vous y référer ultérieurement. Une mauvaise manipulation de ce produit peut entraîner des blessures ou des dommages physiques. Le fabricant n'assume aucune responsabilité pour tout dommage causé par une mauvaise manipulation dépassant l'utilisation normale définie dans ce manuel.
+
+## Avertissements
+
+- <h3><img src="./Img/warning.png" width="20"/> Ne jamais démonter ou modifier, la modification et/ou le désassemblage du produit peut entraîner un choc électrique ou tout autre type de blessure.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Faire preuve d'une prudence particulière à l'égard des enfants et des animaux domestiques, notamment dans les foyers où ils sont présents.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Soyez prudent lorsque vous chargez l'appareil, vous risquez de provoquer un incendie ou un choc électrique. N'appliquez pas trop de chaleur, de pression ou de tension au cordon d'alimentation.</h3>
+
+- <h3><img src="./Img/warning.png" width="20"/> Ne fixez rien d'autre que les éléments spécifiés à l'appareil, toute négligence pourrait entraîner des blessures ou des dommages.</h3>
+
+<div style="page-break-after: always;"></div>
+
+# Premières Etapes
+
+SportShield est un dispositif antivol innovant conçu pour les équipements sportifs, équipé de capteurs et d'actionneurs qui détectent les mouvements, déclenchent une alarme et avertissent le propriétaire.
```

Comment:
You need to change your verb "déclenchent" to "déclenche". It's the SportShield device.